src/js/edu-benefits/education-wizard.js

```diff
@@ -0,0 +1,73 @@
+  function toggleClass(element, className) {
+    element.classList.toggle(className);
+  }
+  function openState(element) {
+    element.setAttribute('data-state', 'open');
+  }
+  function closeState(element) {
+    element.setAttribute('data-state', 'closed');
+    const selectionElement = element.querySelector('input:checked');
+    if (selectionElement) {
+      selectionElement.checked = false;
+    }
+  }
+  function closeStateAndCheckChild(alternateQuestion, container) {
+    const selectionElement = alternateQuestion.querySelector('input:checked');
+    if (!selectionElement) {
+      return closeState(alternateQuestion);
+    }
+    const descendentQuestion = selectionElement.dataset.nextQuestion;
+    if (!descendentQuestion) {
+      return closeState(alternateQuestion);
+    }
+    const descendentElement = container.querySelector(`[data-question=${descendentQuestion}]`);
+    closeStateAndCheckChild(descendentElement, container);
+    return closeState(alternateQuestion);
+  }
+  function reInitWidget() {
+    const container = document.querySelector('.wizard-container');
+    const button = container.querySelector('.wizard-button');
+    const content = container.querySelector('.wizard-content');
+    const radios = container.querySelectorAll('input');
+    const apply = container.querySelector('#apply-now-button');
+    const warning = container.querySelector('#transfer-warning');
+    radios.forEach(radio => {
+      radio.addEventListener('change', () => {
```

Comment:
Idle curiosity: @jbalboni Would your suggestion reduce the memory overhead by having only one event listener?